libs/librepcb/editor/library/pkg/packageeditorwidget.cpp

```diff
@@ -171,9 +171,15 @@ PackageEditorWidget::PackageEditorWidget(const Context& context,
 }
 
 PackageEditorWidget::~PackageEditorWidget() noexcept {
+  // Clean up the state machine nicely to avoid unexpected behavior.
+  mFsm->processAbortCommand();
+  mFsm->processAbortCommand();
+  mFsm->processAbortCommand();
```

Comment:
I just skimmed over the changes, but: Why 3x the same abort call? Is 3 a specific number that is needed? Maybe a loop with an explaining loop count variable name might be a better solution?

Reply:
This abort call does the same as the 'Esc' key, but in some "nested" editor states you need to press 'Esc' several times to completely exit the whole tool. Added a comment to clarify this :)